content/en/spec/activitypub.md

```diff
@@ -218,82 +222,241 @@ alsoKnownAs
 
-- dc:creator
-- dc:created
-- sec:signature
-- sec:signatureValue
+Used for Linked Data Signatures. See [Security > Linked Data Signatures]({{< relref "spec/security#ld" >}}) for more information.
 
-### schema.org extensions (`schema:`) {#schema}
+- [dc:creator (`http://purl.org/dc/terms/creator`)](http://purl.org/dc/terms/creator)
+- [dc:created (`http://purl.org/dc/terms/created`)](http://purl.org/dc/terms/created)
+- [sec:signature (`https://w3id.org/security#signature`)](https://w3id.org/security#signature)
+- [sec:signatureValue (`https://w3id.org/security#signatureValue`)](https://w3id.org/security#signatureValue)
 
-Contains properties used for profile metadata.
+## Extensions defined using ActivityStreams vocabulary
+
+While the Activity Vocabulary defines a wide range of types and terms, ActivityPub only defines side effects for a subset of them. The following activity types have the following side effects when received in a Mastodon inbox.
+
+### Remote blocking (`Block`) {#Block}
+
+ActivityPub defines the `Block` activity for client-to-server (C2S) use-cases, but not for server-to-server (S2S) -- it recommends that servers SHOULD NOT deliver Block activities to their `object`. However, Mastodon will send this activity when a local user blocks a remote user. When Mastodon receives a `Block` activity where the `object` is an actor on the local domain, it will interpret this as a signal to hide the actor's profile and posts from the local user, as well as disallowing mentions of that actor by that local user.
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "id": "https://mastodon.example/bd06bb61-01e0-447a-9dc8-95915db9aec8",
+  "type": "Block",
+  "actor": "https://mastodon.example/users/alice",
+  "object": "https://example.com/~mallory",
+  "to": "https://example.com/~mallory"
+}
+```
+
+### Reporting profiles and posts (`Flag`) {#Flag}
+
+To report profiles and/or posts on remote servers, Mastodon will send a `Flag` activity from the instance actor. The `object` of this activity contains the user being reported, as well as any posts attached to the report. If a comment is attached to the report, it will be used as the `content` of the activity.
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "id": "https://mastodon.example/ccb4f39a-506a-490e-9a8c-71831c7713a4",
+  "type": "Flag",
+  "actor": "https://mastodon.example/actor",
+  "content": "Please take a look at this user and their posts",
+  "object": [
+    "https://example.com/users/1",
+    "https://example.com/posts/380590",
+    "https://example.com/posts/380591"
+  ],
+  "to": "https://example.com/users/1"
+}
+```
+
+### Account migration (`Move`) {#Move}
+
+Mastodon uses the Move activity to signal that an account has migrated to a different account. For the migration to be considered valid, Mastodon checks that the old account has defined an alias pointing to the new account (via the `as:alsoKnownAs` extension property).
```

Comment:
That is the other way around, Mastodon checks that the new account has defined an alias pointing to the old account.

Reply:
oh, oops, you're right. (also note to self, `as:alsoKnownAs` is no longer an "extension" and has been officially adopted)